### ISSUE TITLE
Change the default doc page layout 

### ISSUE THREAD

**Issue by MVrachev** [vmware/precaution#243]
**Is your feature request related to a problem? Please describe.**
We have documentation for both developers/contributors and users in the same place.
It will really valuable and more understandable if we separate them into two groups.

**Describe the solution you'd like**
There can be many ways to separate them and it will be really great if it's visually appealing.

**Describe alternatives you've considered**
A simple solution will be to just separate them into two lists instead of one.
But aren't there other beautiful ways to do this?

**Comment by Stanislava101** [comment]
If no one is working on this issue I can separate them.

**Comment by MVrachev** [comment]
Yes, that would be great. There is no one working on this issue at least from what I know.
